src/app/dataset/preview/preview.component.ts

```diff
@@ -72,12 +73,12 @@ export class PreviewComponent implements OnInit, OnDestroy {
   filterPluginOpen = false;
   historyVersions: Array<HistoryVersion>;
   selectedDate: string;
-  selectedPlugin?: string;
+  selectedPlugin?: PluginType;
   selectedComparison?: string;
   expandedSample?: number;
   nosample: string;
   notification?: Notification;
-  execution: WorkflowExecution;
+  filtered_execution_id: string;
```

Comment:
Why is this separated by underscores, rather than camel-cased?

Reply:
Valid point